hyph-utf8/tex/generic/hyph-utf8/patterns/tex/hyph-la.tex

```diff
@@ -1,43 +1,74 @@
-% of the LaTeX Project Public License Distributed from CTAN
-% archives in directory macros/latex/base/lppl.txt; either
-% version 1 of the License, or any later version.
-% -----------------------------------------------------------------
-%
+% title: Latin Hyphenation Patterns
+% copyright: Copyright (c) 1999-2016 Claudio Beccari
+%                                    e-mail claudio dot beccari at gmail dot com
+% notice: This file is part of the hyph-utf8 package.
+%     See http://www.hyphenation.org for more information.
+% language:
+%     name: Latin
+%     tag: la
+% version: 3.2a <2014/06/04>
+% licence:
+%     - This file is available under any of the following licences:
+%     -
+%         name: MIT
+%         url: https://opensource.org/licenses/MIT
+%         text: >
+%             Permission is hereby granted, free of charge, to any person
+%             obtaining a copy of this software and associated documentation
+%             files (the “Software”), to deal in the Software without
+%             restriction, including without limitation the rights to use,
+%             copy, modify, merge, publish, distribute, sublicense, and/or sell
+%             copies of the Software, and to permit persons to whom the
+%             Software is furnished to do so, subject to the following
+%             conditions:
+%
+%             The above copyright notice and this permission notice shall be
+%             included in all copies or substantial portions of the Software.
+%
+%             THE SOFTWARE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND,
+%             EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES
+%             OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+%             NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+%             HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY,
+%             WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+%             FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+%             OTHER DEALINGS IN THE SOFTWARE.
+%     -
+%         name: LPPL
+%         version: 1
+%         or_later: true
+%         url: https://latex-project.org/lppl/
+% changes:
+%     -
+%         date: 1999
+%         version: 1.0
+%         author: Claudio Beccari
+%         description: First public release
+%     -
+%         date: 2016-08-28
+%         version: 3.201
+%         author: Claudio Beccari
+%         description: updated header with MIT licence notice;
+%                   added few missing patterns
+%
+% ==========================================
 % Patterns for the latin language mainly in modern spelling
 % (u when u is needed and v when v is needed); medieval spelling
-% with the ligatures \ae and \oe  and the (uncial) lowercase `v'
-% written as a `u' is also supported; apparently there is no conflict
+% with the ligatures æ and œ  and the (uncial) lowercase ‘v’
```

Comment:
I'm OK with using unicode characters for digraphs, but generally (applies to all patterns) I would be much more happy to use simple quotation marks.